src/Components/Config/ConfigScreens/AccountsScreen.js

```diff
@@ -65,7 +69,25 @@ class AccountsScreen extends Component {
   render () {
     return (
       <div>
-        <h2>ACCOUNT & KEYS</h2>
+        <h2>ACCOUNTS &amp; KEYS</h2>
+        <section>
+          <h4>ACCOUNT DEFAULT BALANCE</h4>
+          <div className="Row">
+            <div className="RowItem">
+              <input
+                name="server.default_balance_ether"
+                type="text"
+                value={this.props.settings.server.default_balance_ether}
+                onChange={this.validateChange}
+              />
+              {this.props.validationErrors["server.default_balance_ether"] &&
+                <p className="ValidationError">Must be a valid number that is at least {VALIDATIONS["server.default_balance_ether"].min}</p>}
+            </div>
+            <div className="RowItem">
+              <p>Number of Ether that created accounts should be initialized with.</p>
```

Comment:
Small nitpick - this feels a little awkward to me, probably because it ends in a preposition. I'd say something like "The starting balance for accounts, in Ether."